config/lambda/config.go

```diff
@@ -61,6 +62,14 @@ func Configure(p *config.Provider) {
 			SelectorFieldName: "SubnetIDSelector",
 		}
 		delete(r.TerraformResource.Schema, "filename")
+
+		r.TerraformCustomDiff = func(diff *terraform.InstanceDiff, _ *terraform.InstanceState, _ *terraform.ResourceConfig) (*terraform.InstanceDiff, error) {
+			if diff != nil && diff.Attributes != nil {
+				delete(diff.Attributes, "source_code_hash")
+				delete(diff.Attributes, "last_modified")
```

Comment:
In our generated API, `last_modified` is an [observation field](https://github.com/crossplane-contrib/provider-upjet-aws/blob/d29baaf5c73aacdcc9145990a9e2e19f7be6ef9e/apis/lambda/v1beta1/zz_function_types.go#L302), i.e., an attribute. So it should already be not considered in drift calculation (it's not part of the desired state).